README.md

```diff
@@ -21,40 +21,42 @@ Usage: auto-changelog [options]
 
 Options:
 
-  -o, --output [file]                 # output file, default: CHANGELOG.md
-  -c, --config [file]                 # config file location, default: .auto-changelog
-  -t, --template [template]           # specify template to use [compact, keepachangelog, json], default: compact
-  -r, --remote [remote]               # specify git remote to use for links, default: origin
-  -p, --package                       # use version from package.json as latest release
-  -v, --latest-version [version]      # use specified version as latest release
-  -u, --unreleased                    # include section for unreleased changes
-  -l, --commit-limit [count]          # number of commits to display per release, default: 3
-  -b, --backfill-limit [count]        # number of commits to backfill empty releases with, default: 3
-      --commit-url [url]              # override url for commits, use {id} for commit id
-      --issue-url [url]               # override url for issues, use {id} for issue id
-      --merge-url [url]               # override url for merges, use {id} for merge id
-      --compare-url [url]             # override url for compares, use {from} and {to} for tags
-      --issue-pattern [regex]         # override regex pattern for issues in commit messages
-      --breaking-pattern [regex]      # regex pattern for breaking change commits
-      --merge-pattern [regex]         # add custom regex pattern for merge commits
-      --ignore-commit-pattern [regex] # pattern to ignore when parsing commits
```

Comment:
this makes it a breaking change.

Reply:
Im sorry I didn't understand this one, what is the issue?

Reply:
you're removing `--ignore-commit-pattern`

Reply:
yea so it is replaced with `--exclude-commit-pattern`, isnt it?

Reply:
just for consistency, `include` and `exclude`

Reply:
Yes, but removing anything is a breaking change.

Reply:
ah I see, so what do you suggest? going back with `-- ignore-commit-pattern` ?

Reply:
Probably that's simplest, yes.

Reply:
sure